docs/external-id/direct-federation.md

```diff
@@ -108,6 +108,10 @@ Currently, the Microsoft Entra SAML/WS-Fed federation feature doesn't support se
 
 You need to be an [External Identity Provider Administrator](~/identity/role-based-access-control/permissions-reference.md#external-identity-provider-administrator) or a [Global Administrator](~/identity/role-based-access-control/permissions-reference.md#global-administrator) in your Microsoft Entra tenant to configure a SAML/Ws-Fed identity provider.
 
+**Can we avoid the creation of guest/B2B accounts on our tenant?**
+
+Guests accounts will always be need to be created on the teant in order to grant access to applications, roles or membership to security groups or teams in the teanat, regardless of the federation method used.   
```

Comment:
No. A guest account is created for a B2B collaboration user in your directory regardless of the authentication or federation method used. This user object allows you to grant access to applications, assign roles, and define membership in security groups and teams.